pkg/webhook/patch.go

```diff
@@ -384,13 +385,32 @@ func getPrometheusConfigPatches(pod *corev1.Pod, app *v1beta2.SparkApplication)
 		return nil
 	}
 	patchOps = append(patchOps, *vmPatchOp)
-	portPatchOp := addContainerPort(pod, port, protocol, portName)
-	if portPatchOp == nil {
-		glog.Warningf("could not expose port %d to scrape metrics outside the pod", port)
+	promPortPatchOp := addContainerPort(pod, promPort, promProtocol, promPortName)
+	if promPortPatchOp == nil {
+		glog.Warningf("could not expose port %d to scrape metrics outside the pod", promPort)
 		return nil
 	}
-	patchOps = append(patchOps, *portPatchOp)
+	patchOps = append(patchOps, *promPortPatchOp)
+	return patchOps
+}
+
+func addContainerPorts(pod *corev1.Pod, app *v1beta2.SparkApplication) []patchOperation {
+	var PortMaps []v1beta2.Port
```

Comment:
Local variables should start with a lower-case letter. Can you rename this to `ports`?